src/main/java/web/intro/model/Book.java

```diff
@@ -22,4 +23,7 @@ public class Book {
 	private Double price;
```

Comment:
For price better use BigDecimal 
Annotate fields Column annotation with constraints